vllm/logprobs.py

```diff
@@ -0,0 +1,23 @@
+from dataclasses import dataclass
+from typing import Dict, List, Optional
+
+
+@dataclass
+class Logprob:
```

Comment:
Moved this into a separate file to avoid circular dependencies.

(will add ending file newline in subsequent updated)